index.html

```diff
@@ -47,11 +47,173 @@ <h1 class="h-title">I’m ROBERTO. Glad to see you!</h1>
+  <section  class="card works">
+
+    <img src="img/Snapshoot1.png" alt="a_screentshot" class="pic">
+
+    <div class="text-card">
+      <h2 class="titleh2"> Multi-Post Stories</h2>
+        <ul class="canopy">
+          <li class="cano"><a href="#" class="can">CANOPY</a></li>
+          <li><a href="#">Back End Dev</a></li>
+          <li><a href="#">2015</a></li>
+        </ul>
+
+    <p class="texto">
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+
+    <ul>
+        <li class="tag">
+            <a href="">HTML</a>
+        </li>
+        <li class="tag">
+            <a href="" >CSS</a>
+        </li>
+        <li class="tag">
+            <a href="">JavaScript</a>
+        </li>
+    </ul>
+
+    <a href="#" class="button project">See Project</a>
+     </div>
+
+    
+
+</section>
+<section  class="card works">
+
+  <img src="img/Snapshoot2.png" alt="a_screentshot" class="pic">
+
+  <div class="text-card">
+      <h2 class="titleh2">Tonic</h2>
+      <ul class="canopy">
+        <li class="cano"><a href="#" class="can">CANOPY</a></li>
+        <li><a href="#">Back End Dev</a></li>
+        <li><a href="#">2015</a></li>
+      </ul>
+
+  <p class="text">
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+  </p>
+
+  <ul>
+      <li class="tag">
+          <a href="">HTML</a>
+      </li>
+      <li class="tag">
+          <a href="" >CSS</a>
+      </li>
+      <li class="tag">
+          <a href="">JavaScript</a>
+      </li>
+  </ul>
+
+  <a href="#" class="button project">See Project</a>
+   </div>
+
+  
+
+</section>
+<section  class="card works">
+
+  <img src="img/Snapshoot3.png" alt="a_screentshot" class="pic">
+
+  <div class="text-card">
+    <h2 class="titleh2"> Multi-Post Stories</h2>
+      <ul class="canopy">
+        <li class="cano"><a href="#" class="can">CANOPY</a></li>
+        <li><a href="#">Back End Dev</a></li>
+        <li><a href="#">2015</a></li>
+      </ul>
+
+  <p class="text">
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+  </p>
+
+  <ul>
+      <li class="tag">
+          <a href="">HTML</a>
+      </li>
+      <li class="tag">
+          <a href="" >CSS</a>
+      </li>
+      <li class="tag">
+          <a href="">JavaScript</a>
+      </li>
+  </ul>
+
+  <a href="#" class="button project">See Project</a>
+   </div>
+
+  
+
+</section>
+  <section class="myself-section">
+    <div class="title">
+        <h1 class="h-title">About Myself</h1>
+      </div>
+      <div class="parrafo">
+        <p>Hello I’m a software developer! I can help you build a product , feature or website.
+           Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+             don’t hesitate to contact me.
+        </p>
+      </div>
+
+
+<div class="connect">
+<img src="img/LET’S CONNECT.png" alt="connection image">
```

Comment:
- [ ] I noticed that you included this as an image. However, it needs to be a title and hence should have been added using the (`h3` tag) and styled accordingly so kindly implement this :100: